SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -24,6 +36,12 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 	for (size_t i = 0; i < OpList->op_count; ++i)
 	{
 		Worker_Op* Op = &OpList->ops[i];
+
+		if (IsExternalSchemaOp(Op)) {
```

Comment:
```suggestion
		if (IsExternalSchemaOp(Op))
		{
```

Reply:
Could you make this change over the rest of this PR also please :)

Reply:
Ya my bad, in engineering linter used to save me from caring 😄